.gitignore

```diff
@@ -4,6 +4,7 @@
 /dist
 /tmp
 /out-tsc
+/target
```

Comment:
I think this is a build artifact; let me know if I assumed incorrectly

Reply:
It is!